lessons/associative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,100 @@
+** _Possible wonderings: Could I use different numbers and still get the same outcome for both circles? Could I use different operators and still get the same outcome for both circles?
+Work through the remaining problems with your partner. Be sure to evaluate the circles to confirm if they produce equivalent outcomes._
+- What did you observe about the Associative Property?
+** _Possible responses: In each of the examples, the values were the same but they were grouped differently (in different circles). The Associative Property holds for multiplication and addition, but not subtraction or division._
+
+=== Investigate
+
+[.lesson-instruction]
+Turn to Associative Property Table. How many different ways can you apply the Associative Property? Draw as many different Circles of Evaluation as you can.
+
+Note that each of these Circles can be represented in at least one alternative way, as they all include exclusively addition or multiplication.
+
+=== Synthesize
+
+- For which operations does the Associative Property hold?
+** _The Associative Property holds for multiplication and addition, but not division or subtraction._
+- When applying the Associative Property, did the order of the numbers ever change?
+** _No - when we apply the Associative Property on its own, the order of the numbers should always remain the same._
+- When applying the Associative Property, did you ever change the architecture of the Circles? (For instance, did you ever change “circle-inside-circle-inside-circle” to “two-circles-in-a-circle”?) When was that necessary? Why? How did you know that you were doing a valid Circle transformation?
+
+== Circles, the Associative Property, and Mental Math @duration{20 minutes}
+
+=== Overview
+Students discover how the Associative Property and Circles make mental computation simpler.
+
+=== Launch
+
+[.lesson-instruction]
+- Alecia and Jill are trying to solve this multiplication problem: @math{2.5 \times 4 \times 36}
+- Jill takes out a paper and pencil and is hard at work using the multiplication standard algorithm.
+- Alecia looks at the problem for a few seconds and announces that she knows the answer.
+- What is Alecia’s strategy?
+** _Solicit students' ideas. Share one possible path - involving Circles and Transformations - below._
+
+Alecia envisioned a Circle of Evaluation in her head.
+
+[.embedded, cols="^.^3,^.^1,^.^3", grid="none", stripes="none" frame="none"]
+|===
+
+|@show{(coe '(* (* 2.5 4) 36))}	| &rarr; | @show{(coe  '(* 10 36)) }
+| First, she grouped together the first two factors in their own Circle. |  | Then, she applied Computation as a Circle Transformation.
+
+|===
+
+She then applied Computation as a Circle Transformation. Alecia knows that @math{2.5 \times 4} is @math{10}. She also is confident multiplying by powers of 10. Circles helped her to literally see that the final product is @math{3,642}.
```

Comment:
this suggestion would be more powerful if the numbers were ordered differently to start. 2.5 x 36 x 4, where someone had to notice that it would simplify things by starting with multiplying 2.5 by 4, rather than that being the default left to right order.